great_expectations/data_context/data_context.py

```diff
@@ -381,32 +381,55 @@ def _normalize_store_path(self, resource_store):
                 resource_store["base_directory"] = os.path.join(self.root_directory, resource_store["base_directory"])
         return resource_store
 
-    def get_existing_local_data_docs_sites_urls(self):
-        """Get file urls for all built local data docs."""
-        from great_expectations.data_context.store import FixedLengthTupleFilesystemStoreBackend
-        ge_dir = os.path.abspath(self.root_directory)
-        sites = self._project_config_with_variables_substituted["data_docs_sites"]
-
-        existing_sites = []
-
-        for site_name, site in sites.items():
-            store_backend = site.get("store_backend")
-            store_class = load_class(
-                store_backend.get("class_name"),
-                "great_expectations.data_context.store"
-            )
-            # Only do this for local files
-            if issubclass(store_class, FixedLengthTupleFilesystemStoreBackend):
-                base_dir = store_backend.get("base_directory")
-                data_docs_index = os.path.join(ge_dir, base_dir, "index.html")
-
-                if os.path.isfile(data_docs_index):
-                    existing_sites.append("file://" + data_docs_index)
-        return existing_sites
-
-    def open_data_docs(self):
-        """A stdlib cross-platform way to open a file in a browser."""
-        data_docs_urls = self.get_existing_local_data_docs_sites_urls()
+    def get_existing_local_data_docs_sites_urls(self, resource_identifier=None):
```

Comment:
Why is this "local" at this point?

Reply:
The "local" in the method name carried over from the previous implementation. "local" meant "file://". Agreed - this feature should not be limited to this - will change the name.